libs/ui-components/src/components/modals/ApproveDeviceModal/ApproveDeviceForm.css

```
@@ -0,0 +1,18 @@
+/* Style to get the appearance of a disabled Input with messages spanning multiple lines */
+.fctl_approve-device__fleetname {
+  display: flex;
+  align-items: center;
+  background-color: var(--pf-v5-global--disabled-color--300);
+  padding: 0.5rem;
+}
+
+.fctl_approve-device__fleetname__icon  {
```

sorry I missed this previously. 
The naming does not follow BEM

it should be 
```suggestion
.fctl-approve-device__fleetname-icon  {
```

---

No worries!
Since I split the component to a separate file, the className is simplified and does not have this issue anymore.